lib/chef/knife/opc_user_delete.rb

```diff
@@ -19,126 +19,38 @@
 
 module Opc
   class OpcUserDelete < Chef::Knife
-    category "CHEF ORGANIZATION MANAGEMENT"
-    banner "knife opc user delete USERNAME [-d] [-R]"
-
-    option :no_disassociate_user,
-      long: "--no-disassociate-user",
-      short: "-d",
-      description: "Don't disassociate the user first"
-
-    option :remove_from_admin_groups,
-      long:  "--remove-from-admin-groups",
-      short:  "-R",
-      description: "If the user is a member of any org admin groups, attempt to remove from those groups. Ignored if --no-disassociate-user is set."
```

Comment:
this should not be getting removed, and a bunch of other stuff in here is not part of the other PR which is being rebased.